sdk/python/test_data/test_data_config.yaml

```diff
@@ -80,9 +80,6 @@ pipelines:
           "first_number": 1,
           "second_number": 2,
         }
-    - module: xgboost_sample_pipeline
-      name: xgboost_pipeline
-      execute: false
```

Comment:
We are removing the following content from `test_data_config.yaml` because it is used inside the test suite. The test goes to that YAML, loads the content, and executes a test in runtime.
```
    - module: xgboost_sample_pipeline
      name: xgboost_pipeline
      execute: false
```

The files were removed because:
* they contain references to `components.load_component_from_url` from a path that was removed `components/XGBoost`
* Those references contain references to a Python image that is less than equals to 3.8.
If the files were removed and if the files contain references to a Python image that is not minimal, then we remove them.

Replied here and updated the description of the PR with the same text.